iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/AffMatchingJob.java

```diff
@@ -90,12 +101,37 @@ private static AffMatchingService createAffMatchingService() {
         affMatchingService.setAffiliationReader(new IisAffiliationReader());
         affMatchingService.setOrganizationReader(new IisOrganizationReader());
         
+        DocumentOrganizationReader documentOrganizationReader = new DocumentOrganizationReader();
+        documentOrganizationReader.setDocumentProjectReader(new IisDocumentProjectReader());
+        documentOrganizationReader.setProjectOrganizationReader(new IisProjectOrganizationReader());
+        documentOrganizationReader.setDocumentOrganizationCombiner(new DocumentOrganizationCombiner());
+        documentOrganizationReader.setDocProjConfidenceLevelThreshold(params.inputDocProjConfidenceThreshold);
```

Comment:
We have more and more parameters to be set programatically. It will look much nicer having Spring organizing dependencies.